src/main.dart

```diff
@@ -1,8 +1,193 @@
-// TODO: Run task list app
-void main(List<String> args) {
-  print('Hello World');
+import 'dart:io';
+
+class Task {
+  String description;
+  bool isCompleted;
+
+  Task(this.description, {this.isCompleted = false});
+}
+
+void main() {
+  final List<Task> tasks = [];
+
+  while (true) {
+    print("Task List");
+    print("1. task number.");
+    print("2. Add Title.");
+    print("3. add description.");
+    print("4. edit Task.");
+    print("5. delete task.");
+    print("6. mark task as completed.");
+    print("7. show all tasks.");
+    print("8. exit.");
+
+    String input = stdin.readLineSync() ?? " ";
+    int choice = int.tryParse(input) ?? 0;
+
+    switch (choice) {
+      
+      case 1:
+       addTasknum(tasks);
```

Comment:
install a spell checker extension to ensure proper function, variable, and class naming